### ISSUE TITLE
Differences with official holidays (Spain 2021)

### ISSUE THREAD

**Issue by fidelpita** [nager/Nager.Date#227]
### Which country is affected?

Spain

### The source of the information?

Official State Gazette: [https://www.boe.es/boe/dias/2020/11/02/pdfs/BOE-A-2020-13343.pdf](https://www.boe.es/boe/dias/2020/11/02/pdfs/BOE-A-2020-13343.pdf)

### Description

There are several differences between the official listing of public holidays for 2021 in Spain ([Official State Gazette](https://www.boe.es/boe/dias/2020/11/02/pdfs/BOE-A-2020-13343.pdf)) and the results obtained via Nager.Date ([https://date.nager.at/PublicHoliday/Country/ES/2021](https://date.nager.at/PublicHoliday/Country/ES/2021)).

**Comment by tinohager** [comment]
Source: https://github.com/nager/Nager.Date/blob/master/Src/Nager.Date/PublicHolidays/SpainProvider.cs

**Comment by tinohager** [comment]
@fidelpita Can you summarize the differences?

**Comment by fidelpita** [comment]
The following table summarizes the differences. Please note that some holidays that happen to be on a Sunday (e.g. Saint James, St. Stephen's, Día de Canarias) and have no substitute day do not appear on the list but will presumably reappear on 2022. Although they don't formally appear on the Official State Gazette, perhaps these could still be displayed for 2021 (not sure).

Date | Local name | Name | Status
-- | -- | -- | --
Fr 01/01/2021 | Año Nuevo | New Year's Day | ok
We 06/01/2021 | Día de Reyes / Epifanía del Señor | Epiphany | ok
Su 28/02/2021 | Día de Andalucía | Regional Holiday | ok
Mo 01/03/2021 | Dia de les Illes Balears | Regional Holiday | ok
Fr 19/03/2021 | San José | St. Joseph's Day | To be added: ES-EX, ES-PV; To be removed:   ES-ML, ES-CM, ES-IB, ES-O
Th 01/04/2021 | Jueves Santo | Maundy Thursday | ok
Fr 02/04/2021 | Viernes Santo | Good Friday | ok
Su 04/04/2021 | Pascua de Resurrección | Easter Sunday | To be removed
Mo 05/04/2021 | Lunes de Pascua | Easter Monday | To be added: ES-LO
Fr 23/04/2021 | San Jorge (Día de Aragón) | Regional Holiday | ok
Fr 23/04/2021 | Día de Castilla y León | Regional Holiday | ok
Sa 01/05/2021 | Día del Trabajador | Labour Day | Local name to be changed to “Fiesta del trabajo”
Su 02/05/2021 | Fiesta de la Comunidad de Madrid | Regional Holiday | Substitute: 2021-05-03
Mo 17/05/2021 | Día das Letras Galegas | Regional Holiday | ok
Su 30/05/2021 | Día de Canarias | Regional Holiday | To be removed
Mo 31/05/2021 | Día de la Región Castilla-La Mancha | Regional Holiday | ok
Th 03/05/2021 | Corpus Christi | Corpus Christi | To be added
We 09/06/2021 | Día de la Región de Murcia | Regional Holiday | ok
We 09/06/2021 | Día de La Rioja | Regional Holiday | ok
Th 24/06/2021 | Sant Joan | St. John's Day | To be added: ES-VC
Tu 20/07/2021 | Fiesta del Sacrificio-Eidul Adha | Eidul Adha | To be added
We 21/07/2021 | Fiesta del Sacrificio-Aid Al Adha | Aid Al Adha | To be added
Su 25/07/2021 | Santiago Apóstol | Saint James | To be removed
We 28/07/2021 | Día de las Instituciones de   Cantabria | Regional Holiday | To be added
Su 15/08/2021 | Asunción | Assumption | Substitute: 2021-08-16
Th 02/09/2021 | Día de Ceuta | Municipal Holiday | ok
We 08/09/2021 | Día de Asturias | Regional Holiday | ok
We 08/09/2021 | Día de Extremadura | Regional Holiday | ok
Sa 11/09/2021 | Diada Nacional de Catalunya | National Day of Catalonia | ok
We 15/09/2021 | Día de Cantabria | Regional Holiday | Local name to be changed to: “La Bien Aparecida”
Sa 09/10/2021 | Dia de la Comunitat Valenciana | Regional Holiday | ok
Tu 12/10/2021 | Fiesta Nacional de España | Fiesta Nacional de España | ok
Mo 25/10/2021 | Euskadi Eguna | Regional Holiday | To be removed
Mo 01/11/2021 | Día de todos los Santos | All Saints Day | ok
Mo 06/12/2021 | Día de la Constitución | Constitution Day | ok
We 08/12/2021 | Inmaculada Concepción | Immaculate Conception | ok
Sa 25/12/2021 | Navidad | Christmas Day | ok
Su 26/12/2021 | Sant Esteve | St. Stephen's Day | To be removed
